sdk/python/feast/infra/offline_stores/bigquery.py

```diff
@@ -200,7 +196,7 @@ def __init__(
         client: bigquery.Client,
         config: RepoConfig,
         full_feature_names: bool,
-        on_demand_feature_views: Optional[List[OnDemandFeatureView]],
+        on_demand_feature_views: List[OnDemandFeatureView] = [],
```

Comment:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

This should continue to be `Optional` but you can set it to `[]` in the function body if it's `None`.